docs/docs/reference/impersonation.md

```diff
@@ -32,12 +38,133 @@ In certain circumstances, it's useful for an administrative user to impersonate
 
 :::
 
-## Example
-
 Here's what it looks like.
 
 <video width="100%" height="600" controls=""><source src="./img/pomerium-user-impersonation.mp4" type="video/mp4">
 Your browser does not support the video tag.
 </video>
 
+### Using the command line interface
+
+Pomerium also includes a command line interface (cli) for generating arbitrary route-scoped service account sessions. Generated service accounts can be used to impersonate users, perform service-to-service communication, and facilitate end-to-end testing for applications managed by Pomerium. The cli is especially useful in situations where an administrator needs more control over the sessions she generates, or if skipping the authentication portion of pomerium's flow is required.
+
+### How
+
+```bash
+pomerium-sa generates a pomerium service account from a shared key.
+
+Usage: /bin/pomerium-cli [flags] [base64d shared secret setting]
+
+For additional help see:
+
+    https://www.pomerium.io
+    https://jwt.io/
+
+Flags:
+
+  -aud value
+        Audience (e.g. httpbin.int.pomerium.io,prometheus.int.pomerium.io)
+  -email string
+        Email
+  -expiry duration
+        Expiry (default 1h0m0s)
+  -groups value
+        Groups (e.g. admins@pomerium.io,users@pomerium.io)
+  -impersonate_email string
+        Impersonation Email (optional)
+  -impersonate_groups value
+        Impersonation Groups (optional)
+  -iss string
+        Issuing Server (e.g authenticate.int.pomerium.io)
+  -sub string
+        Subject (typically User's GUID)
+  -user string
+        User (typically User's GUID)
+```
+
+For example, say for example I wanted to generate a service account a session with the following attributes:
+
```

Comment:
tiny suggestion: what do you think about improving the readability here... `For example, I want to generate a service account session with the following attributes:` or anything you think can improve the readability.

Reply:
I've removed the extraneous example text. hopefully it's more clear now.